pom.xml

```diff
@@ -139,10 +139,11 @@
 
     <api-client.version>1.23.0</api-client.version>
 
-    <api-common.version>1.1.0</api-common.version>
-    <gax.version>1.13.0</gax.version>
-    <gax-grpc.version>0.30.0</gax-grpc.version>
-    <generatedProto.version>0.1.23</generatedProto.version>
+    <api-common.version>1.2.0</api-common.version>
+    <gax.version>1.14.0</gax.version>
+    <gax-grpc.version>1.14.0</gax-grpc.version>
+    <generatedProto.version>0.1.24</generatedProto.version>
```

Comment:
Please consider renaming this to `generated-proto.version` and `generated-proto-ga.version` (or even `generated-proto.version` and `generated-proto-beta.version`, to move the ugliness to the beta property name from ga property name)

Reply:
I'll clean this up in a following pass in order to streamline my work today.